### PR TITLE
[14.0] [mig] hr_timesheet_analysis

### PR DESCRIPTION
**Pull request by fshah-initos**

### CODE REVIEW

**Comment by fshah-initos** [comment]
@davejames will you please review this PR?

**Review by dsolanki-initos — APPROVED** [review]
Looks Good to me. Functional test and code review.

**Comment by fshah-initos** [comment]
@alexey-pelykh Will you please review the PR?

**Comment by fshah-initos** [comment]
@bjeficent  can you please review this PR?

**Comment by fshah-initos** [comment]
It would be really great if someone could review the PR.

File: hr_timesheet_analysis/views/account_analytic_line.xml [hunk 1]
```diff
@@ -4,17 +4,6 @@
     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 -->
 <odoo>
-    <record id="hr_timesheet_line_pivot" model="ir.ui.view">
```
Comment (alexey-pelykh) [inline hr_timesheet_analysis/views/account_analytic_line.xml]: Pivots are gone from v14?

Comment (fshah-initos) [reply hr_timesheet_analysis/views/account_analytic_line.xml]: It is alreday available in 14 core so removed it.

**Comment by fshah-initos** [comment]
Hi @alexey-pelykh can you please re-check so this PR can be merged ASAP.

**Comment by fshah-initos** [comment]
It would be great if you could re-check and reviews are added here.

**Comment by fshah-initos** [comment]
It would be great if you could get the reviews here.

**Comment by fshah-initos** [comment]
It would be great if we could merge this PR

**Comment by fshah-initos** [comment]
@OCA/human-resources-maintainers can you please merge this PR?

**Review by Saran440 — APPROVED** [review]
Code no change :+1:

**Pull request merged**